judo-runtime-core-dao-rdbms/src/main/java/hu/blackbelt/judo/runtime/core/dao/rdbms/AbstractRdbmsDAO.java

```diff
@@ -197,6 +197,17 @@ protected Payload create(EClass eClass, Payload payload, QueryCustomizer<ID> que
         }
     }
 
+    @Override
+    public List<Payload> createAll(EClass eClass, Iterable<Payload> payloads, QueryCustomizer<ID> queryCustomizer) {
+        List resultPayloads = new ArrayList<>();
```

Comment:
Metrics collector is missing?

Reply:
CreateAll function call create function, which use MetricsCollector. You can see it in line 192.